### ISSUE TITLE
Event Loop Causes Live Timing Crash

### ISSUE THREAD

**Issue by Hecrer** [cj123/assetto-server-manager#517]
See https://www.racedepartment.com/threads/ac-server-manager.165662/page-17#post-3043213

**Comment by cj123** [comment]
race_control.go lines ~222

```
	if sessionInfo.CurrentSessionIndex != 0 && oldSessionInfo.Track == sessionInfo.Track && oldSessionInfo.TrackConfig == sessionInfo.TrackConfig {
		// only remove cars on the first session (avoid deleting between practice/qualify/race)
		deleteCars = false
		emptyCarInfo = true
	}
```

**Comment by cj123** [comment]
fixed in #592
